i18n/en.json

```diff
@@ -1256,6 +1256,10 @@
     "id": "api.lti.signup.no_lms_found",
     "translation": "There was an error with LTI signup attempt. Please try again or contact your system administrator. Error code LTI-12"
   },
+  {
+    "id": "onboard_lms_user.team_not_found.app_error",
```

Comment:
Since this is an `App` function, it can be called from anywhere, not limited to API. Additionally, since this is not an API function, I don't think it makes sense.